src/administration/integrations.md

```diff
@@ -26,16 +26,92 @@ platform integrations
+
+```bash
+$ platform integration:activities
+
+Enter a number to choose an integration:
+  [0] dxr45hfldrkoe (webhook)
+  [1] n2ukd4p7qofg4 (health.email)
+  [2] c4opi5tjv3yfd (github)
+ > 2
+
+Activities on the project Platform.sh | Docs (6b2eocegfkwwg), integration c4opi5tjv3yfd (github):
++---------------+---------------------------+-------------------------------------------------------------+----------+---------+
+| ID            | Created                   | Description                                                 | State    | Result  |
++---------------+---------------------------+-------------------------------------------------------------+----------+---------+
+| 6456zmdtoykxa | 2020-04-14T16:38:09-05:00 | Fetching from https://github.com/platformsh/platformsh-docs | complete | success |
+| wcwp34yjvydgk | 2020-04-14T16:35:22-05:00 | Fetching from https://github.com/platformsh/platformsh-docs | complete | success |
+| w2bp3oa5xbfoe | 2020-04-14T16:33:13-05:00 | Fetching from https://github.com/platformsh/platformsh-docs | complete | success |
+| uqqvdyxmcdmsa | 2020-04-14T16:31:45-05:00 | Fetching from https://github.com/platformsh/platformsh-docs | complete | success |
+| 7x3wefhh4fwqc | 2020-04-14T16:30:36-05:00 | Fetching from https://github.com/platformsh/platformsh-docs | complete | success |
+| a46aah3ga65gc | 2020-04-14T16:29:46-05:00 | Fetching from https://github.com/platformsh/platformsh-docs | complete | success |
+| r7erid2jlixgi | 2020-04-14T16:24:50-05:00 | Fetching from https://github.com/platformsh/platformsh-docs | complete | success |
+| ieufk3vvde5oc | 2020-04-14T16:24:49-05:00 | Fetching from https://github.com/platformsh/platformsh-docs | complete | success |
+| bc7ghg36ty4ea | 2020-04-14T15:30:17-05:00 | Fetching from https://github.com/platformsh/platformsh-docs | complete | success |
+| 4qojtv7a6rk2w | 2020-04-14T15:27:26-05:00 | Fetching from https://github.com/platformsh/platformsh-docs | complete | success |
++---------------+---------------------------+-------------------------------------------------------------+----------+---------+
+```
+
+You may also specify which integration to display in the command line directly: `platform integration:activities c4opi5tjv3yfd`.
+
+The ID is an internal identifier for the activity event.  The Description field is an arbitrary string of text produced by the integration code.  The State and Result fields indicate if the activity completed successfully, failed for some reason, or is currently in progress.
+
+See the `--help` output of the command for more options.
+
+### Showing detailed information on an activity
+
+You can call up more detailed information on a specific activity by its ID, using the `platform integration:activity:get` command.  It requires both the integration ID and an activity ID from the list above.
+
+```bash
+$ platform integration:activity:get c4opi5tjv3yfd 4qojtv7a6rk2w
+Environment ID [master]:
```

Comment:
lines 92 and 93 are a bug that are being fixed by https://github.com/platformsh/platformsh-cli/commit/fbd16b33335dcba21f7b30d365b72f8698868bb3